.github/workflows/docs.yaml

```diff
@@ -45,7 +45,8 @@ jobs:
           sudo apt-get install pandoc dvisvgm texlive texlive-latex-extra
       - name: Build documentation
         run: make docs
-
+      - name: Validate examples
+        run: python3 ./.github/scripts/examples_validation.py
```

Comment:
why is it under .github and not tests?

Reply:
Thought it better to have it there since it's an action logic. but I'll move it